### ISSUE TITLE
tecff-ath9k-broken-wifi: 035-cronloglevel: Permission denied

### ISSUE THREAD

**Issue by krombel** [freifunkMUC/gluon-packages#7]
```
# gluon-reconfigure
[...]
Configuring: 035-cronloglevel
/usr/bin/gluon-reconfigure: line 10: ./035-cronloglevel: Permission denied
[...]
# ls -l /lib/gluon/upgrade/035-cronloglevel 
-rw-r--r--    1 root     root           178 Feb 25 20:42 /lib/gluon/upgrade/035-cronloglevel
```
This script does not run on gluon-v2018.2 anymore as it requires 'luci.model.uci'. So it needs to get updated if we want to use it

**Comment by rotanid** [comment]
i would suggest to update every package you use from external source, that way you would already have many fixes for this package tecff-ath9k-....
it doesn't make sense to only fix one tiny thing locally.

**Comment by krombel** [comment]
fixed in #12
